src/_scss/pages/covid19/index.scss

```diff
@@ -22,65 +21,186 @@
 
     @import "../explorer/detail/sidebar/fyPicker";
 
-    @import "./stickyHeader";
+    .sticky-header .sticky-header__container .sticky-header__header {
```

Comment:
Could this move to the new `./_stickyHeader.scss` file? Trying to keep `index.scss` from becoming a massive file!

Reply:
Updated!